README.md

```diff
@@ -36,6 +36,9 @@ ec2-user hard nproc 16384
 EOL
 ```
 
+Note: In the above configuration, `ec2-user` it's a
```

Comment:
english-wise you'd say " `ec2-user` **is** a placeholder..."

Reply:
Sure. My mistake. I'll update.